### ISSUE TITLE
Remove a sha1 import from < 2.5

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5864]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#5864|
|Type|task|
|Created|2012-08-14 14:59:47Z|
|Branch|https://github.com/twisted/twisted/tree/sha1-py25-5864|

Simple removal of an import of `sha1` from pre-2.5.

Attachments:

* [knownhosts-24.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/548/54831f4b334638d96ebb960a5f69cd4c08db91af/134d6e42ce8e984088347478ed471833cadf4c4d.patch) (568 bytes) - added by Julian on 2012-08-14 15:00:19Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5864 5864
type__task task
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_sha1_py25_5864 branches/sha1-py25-5864
branch_author__Julian Julian
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1344956387000000 1344956387000000
changetime__1360620500000000 1360620500000000
version__None None
owner__tom_prince tom.prince
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_2"></a>|@vperic set owner to @Julian|
|-|-|

Looks good to me.

Also, while I think small tickets are great, this is *really* small. Perhaps it would be better to work on tickets [#5386](https://github.com/twisted/twisted/issues/5386) and [#5837](https://github.com/twisted/twisted/issues/5837), which are about this but for the whole of Twisted. Grouping it by module might also work well.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_3"></a>|@Julian commented|
|-|-|

Hey, thanks for the review :).

These three were small fixes that were just easy enough to do while I had these modules open for another reason, I've got a couple of other larger issues that I've been working on.

I don't have merge rights I don't think, so if you happen to and have a moment that'd be nice :). Thanks again.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [37019]) Branching to 'sha1-py25-5864'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner|
|-|-|

Forced a [build](https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/sha1-py25-5864).

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_6"></a>|@tomprince set owner to @tomprince|
|-|-|

Merging: build looks clean, and vperic already gave this a good review.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_7"></a>|@tomprince set status to `closed`|
|-|-|

(In [37119]) Merge sha1-py25-5864: Simple removal of an import of sha1 from pre-2.5.

Author: Julian
Reviewers: vperic
Fixes: [#5864](https://github.com/twisted/twisted/issues/5864)
